tools/run_all_tests.py

```diff
@@ -126,6 +129,11 @@ def test_all(
                 break
         else:
             raise ValueError(f"Test to skip '{test_to_skip}' not found in tests.")
```

Comment:
Not exactly related but is there a typo here: https://github.com/isaac-sim/IsaacLab/blob/3815c6a554594ce0c4160be58554718d4a4efd9b/tools/run_all_tests.py#L129-L131

Reply:
Where is the typo?

Reply:
The else case is outside the if?

Reply:
Ah nevermind. Didn't know about the for-else semantics 😅